Source/Models/YPMediaItem.swift

```diff
@@ -47,6 +47,8 @@ public class YPMediaVideo {
     // used to defer clipping and cropping until necessary
     public var timeRange: CMTimeRange?
     public var cropRect: CGRect?
+    public var startTime: CMTime?
```

Comment:
Are these two values effectively the same as `timeRange` above? (I'm not sure, but it seems like `timeRange` might model the same thing, just in a clumsier format)

Reply:
Yea, looks like I can pull start and duration from that. I'll give it a go

Reply:
Done!